src/clr-angular/modal/modal.html

```diff
@@ -31,11 +31,6 @@
                     <ng-content select=".modal-footer"></ng-content>
                 </div>
             </div>
-            <!--todo: deprecate the modal-ghost-wrapper div below after 0.12-->
-            <div class="modal-ghost-wrapper">
-                <div [@ghostPageOneState]="ghostPageState" class="modal-ghost modal-ghost-1"></div>
-                <div [@ghostPageTwoState]="ghostPageState" class="modal-ghost modal-ghost-2"></div>
-            </div>
```

Comment:
i thought the DOM edits would be more severe than this. i wonder if the outer-wrapper is still necessary?

Reply:
I'll investigate if the removal of outer-wrapper can be done without breaking anything.